Modules/Microsoft365DSC/Examples/Resources/EXOActiveSyncMailboxPolicy/3-Remove.ps1

```diff
@@ -0,0 +1,89 @@
+<#
+This example is used to test new resources and showcase the usage of new resources being worked on.
+It is not meant to use as a production baseline.
+#>
+
+Configuration Example
+{
+    param(
+        [Parameter()]
+        [System.String]
+        $ApplicationId,
+
+        [Parameter()]
+        [System.String]
+        $TenantId,
+
+        [Parameter()]
+        [System.String]
+        $CertificateThumbprint
+    )
+    Import-DscResource -ModuleName Microsoft365DSC
+
+    node localhost
+    {
+        EXOActiveSyncMailboxPolicy 'TestActiveSyncMailboxPolicy'
+        {
+            AllowApplePushNotifications              = $True;
+            AllowBluetooth                           = "Allow";
+            AllowBrowser                             = $True;
+            AllowCamera                              = $True;
+            AllowConsumerEmail                       = $True;
+            AllowDesktopSync                         = $True;
+            AllowExternalDeviceManagement            = $False;
+            AllowHTMLEmail                           = $True;
+            AllowInternetSharing                     = $True;
+            AllowIrDA                                = $True;
+            AllowMobileOTAUpdate                     = $True;
+            AllowNonProvisionableDevices             = $True;
+            AllowPOPIMAPEmail                        = $True;
+            AllowRemoteDesktop                       = $True;
+            AllowSimpleDevicePassword                = $True;
+            AllowSMIMEEncryptionAlgorithmNegotiation = "AllowAnyAlgorithmNegotiation";
+            AllowSMIMESoftCerts                      = $True;
+            AllowStorageCard                         = $True;
+            AllowTextMessaging                       = $True;
+            AllowUnsignedApplications                = $True;
+            AllowUnsignedInstallationPackages        = $True;
+            AllowWiFi                                = $True;
+            AlphanumericDevicePasswordRequired       = $False;
+            ApprovedApplicationList                  = @();
+            AttachmentsEnabled                       = $True;
+            Credential                               = $Credscredential;
```

Comment:
Credentials is not passed as a parameter. We need only appId, tenantId and certThumbprint

Reply:
Removed